enterprise/cluster/src/main/java/org/neo4j/cluster/protocol/atomicbroadcast/multipaxos/LearnerMessage.java

```diff
@@ -50,6 +53,18 @@ public Object getValue()
         @Override
         public String toString()
         {
+            if (value instanceof Payload )
+            {
+                try
+                {
+                    ObjectStreamFactory streamFactory = new ObjectStreamFactory();
+                    return new AtomicBroadcastSerializer( streamFactory, streamFactory ).receive( (Payload) value).toString();
```

Comment:
To avoid duplication of code, why don't move this logic into `Payload.toString`?